tests/e2e/specs/login_spec.js

```diff
@@ -56,21 +55,22 @@ describe('Login', () => {
     cy.checkCustomerIsLoggedIn(customer);
   });
 
-  it('resets password', () => {
-    cy.createCustomer(newCustomer);
-    cy.visit('/forgot-password');
-    cy.get('[data-test=forgot-password-email]').type(newCustomer.email);
-    cy.get('[data-test=forgot-password-form-submit]').click();
-    cy.wrap(mailslurp.waitForEmailCount(1, msEmail.substring(0, msEmail.lastIndexOf('@')))
-      .then((response) => mailslurp.getEmail(response[0].id)
-        .then((fullEmail) => fullEmail.body.match(/a href="([^"]*)/)[1])))
-      .then((link) => {
-        cy.visit(link);
-        cy.get('[data-test=reset-new-password]').type(newPassword);
-        cy.get('[data-test=reset-confirm-password]').type(newPassword);
-        cy.get('[data-test=reset-password-submit]').click();
-        cy.login({ email: newCustomer.email, password: newPassword });
-        cy.checkCustomerIsLoggedIn(newCustomer);
-      });
-  });
+  // TODO: fix password reset serverless function
+  // it('resets password', () => {
```

Comment:
@plaskevich  You can change to xit, no need to comment it out. I did the same with [my branch](https://github.com/commercetools/sunrise-spa/pull/227/files/a2b1759f0fdd26e076c0a6c2d9ce48e1c8f0d4d5#diff-9a1b9717662fe63842b7d391fe30afd5e08fa8f6d4aa4e0a7def714558d127f3). This will cause a conflict.

Reply:
Will fix this in later commit.